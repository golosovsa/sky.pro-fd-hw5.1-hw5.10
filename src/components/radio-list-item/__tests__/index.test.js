import ShallowRenderer from 'react-test-renderer/shallow'
import RadioItem from '..'

describe('<RadioList />', () => {
  describe('Snapshot test', () => {
    it('should render successfully and match snapshot', () => {
      const renderer = new ShallowRenderer()
      const tree = renderer.render(<RadioItem checked={false}>children</RadioItem>)

      expect(tree).toMatchSnapshot()
    })

    it('should render successfully and match snapshot is checked', () => {
      const renderer = new ShallowRenderer()
      const tree = renderer.render(<RadioItem checked={true}>children</RadioItem>)

      expect(tree).toMatchSnapshot()
    })
  })
})
