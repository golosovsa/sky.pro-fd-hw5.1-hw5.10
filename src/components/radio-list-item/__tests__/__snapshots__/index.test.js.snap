// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RadioList /> Snapshot test should render successfully and match snapshot 1`] = `
<styled.div>
  <styled.div />
  children
</styled.div>
`;

exports[`<RadioList /> Snapshot test should render successfully and match snapshot is checked 1`] = `
<styled.div>
  <styled.div>
    <styled.div />
  </styled.div>
  children
</styled.div>
`;
